cmd/nerdctl/builder_build.go

```diff
@@ -51,6 +51,7 @@ If Dockerfile is not present and -f is not specified, it will look for Container
 	buildCommand.Flags().StringP("output", "o", "", "Output destination (format: type=local,dest=path)")
 	buildCommand.Flags().String("progress", "auto", "Set type of progress output (auto, plain, tty). Use plain to show container output")
 	buildCommand.Flags().String("provenance", "", "Shorthand for \"--attest=type=provenance\"")
+	buildCommand.Flags().String("pull", "", "On true, always attempt to pull latest image version from remote. Default uses buildkit's default.")
```

Comment:
Can we make it a bool flag and just check if it's set when parsing?

Reply:
Is this possible? From my understanding, the bool flag can only be true or false, and if none are provided then it will just set a default value.

Once parsed from a string, though, I can store it into a *bool instead of a string.

Reply:
Have you tried `Changed`? https://pkg.go.dev/github.com/spf13/pflag#FlagSet.Changed

Reply:
This is perfect, thanks. Just pushed a new set of changes.